spec/game_spec.rb

```diff
@@ -6,4 +6,21 @@
         pass = true
         expect(pass).to eq(true)
    end
+
```

Comment:
Can we also add a test here for printing `Welcome to Tic-Tac-Toe` upon Game initialization?